packages/twenty-front/src/modules/settings/page-layout/utils/graphRegistry.tsx

```diff
@@ -3,9 +3,10 @@ import { GraphWidgetGaugeChart } from '@/dashboards/widgets/graph/components/Gra
 import { GraphWidgetNumberChart } from '@/dashboards/widgets/graph/components/GraphWidgetNumberChart';
 import { GraphWidgetPieChart } from '@/dashboards/widgets/graph/components/GraphWidgetPieChart';
 import { type ReactNode } from 'react';
-import { GraphSubType, type Widget } from '../mocks/mockWidgets';
+import { GraphSubType } from '../mocks/mockWidgets';
+import { type PageLayoutWidget } from '../states/savedPageLayoutsState';
 
-type GraphRenderer = (widget: Widget) => ReactNode;
+type GraphRenderer = (widget: PageLayoutWidget) => ReactNode;
```

Comment:
this is non standard to have an util which is .tsx I think we need a component here